project-api/project-api-test/src/test/java/io/ballerina/projects/test/HierarchicalPackageNameTests.java

```diff
@@ -134,7 +134,7 @@ public void testResolveHierarchicalPackageInDist() {
             if (importModuleResponse.resolutionStatus().equals(ResolutionResponse.ResolutionStatus.UNRESOLVED)) {
                 Assert.assertEquals(importModuleResponse.importModuleRequest(), request4);
             } else if (importModuleResponse.importModuleRequest().moduleName().equals("a.c")) {
-                Assert.assertEquals(importModuleResponse.packageDescriptor().name().toString(), "a");
+                Assert.assertEquals(importModuleResponse.packageDescriptor().name().toString(), "a.c");
```

Comment:
Are these unrelated changes to tests required?

Reply:
Updated